addons/website_sale/views/templates.xml

```diff
@@ -1305,11 +1305,23 @@
     </template>
 
     <template id="website_sale.tax_indication" active="False">
-        <span t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'" class="h6 text-muted">
-            VAT Excluded
-        </span>
-        <span t-else="" class="h6 text-muted">
-            VAT Included
+        <span class="h6 text-muted">
+            <t t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'">
+                <t t-if="website.company_id.country_code == 'IN'">
```

Comment:
Hello @mish-odoo 

I believe that this should be in a separate module `l10n_in_website_sale` as it's a tax only applicable in India.